plugins/modules/mso_schema_site_external_epg.py

```diff
@@ -160,7 +214,17 @@ def main():
             ops.append(dict(op="remove", path=op_path))
 
     elif state == "present":
-        l3out_dn = "uni/tn-{0}/out-{1}".format(tenant_name, l3out)
+        # Get external EPGs type from template level
+        external_epgs = [e.get("name") for e in schema_obj.get("templates")[template_idx]["externalEpgs"]]
```

Comment:
do we need this list when external_epg is None? should we move logic inside if statement?